### PR TITLE
RSKIP-412 for basefee opcode

### PR DESCRIPTION
**Pull request by Vovchyk**
we want to introduce a new instruction that would return a block minimum gas price.

### CODE REVIEW

File: IPs/RSKIP412.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# **Abstract**
+
+This RSKIP implements [Ethereum BASEFEE instruction](https://eips.ethereum.org/EIPS/eip-3198). The `BASEFEE (0x48)` opcode returns the base fee value of the current block it is executing in.
+
+# **Motivation**
+
+In order to maintain compatibility with the EVM, RSK needs to implement the BASEFEE opcode.
+
+There's an important difference when comparing *RSK* vs *ETH* `BASEFEE` opcode: as RSK doesn't define a base fee for its blocks, the block `minimum gas price` is returned instead. More details on the specification section.
```
Comment (aeidelman) [inline IPs/RSKIP412.md:34]: Please use "Rootstock" instead of "RSK"

File: IPs/RSKIP412.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+# **Specification**
+
+`BASEFEE` opcode pushes on the stack the minimum gas price of the block where the transaction is executing. The cost of `BASEFEE` is 2.
```
Comment (aeidelman) [inline IPs/RSKIP412.md:38]: Some minor corrections: "The BASEFEE opcode pushes onto the stack the minimum gas price of the block in which the transaction is being executed. The cost of the BASEFEE opcode is 2."

File: IPs/RSKIP412.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+rskip: 412
+title: BASEFEE instruction
+description:
+status: Draft
+purpose: Usa
+author: VK
+layer: Core
+complexity: 2
+created: 2023/11/
```
Comment (josedahlquist) [inline IPs/RSKIP412.md:10]:
```suggestion
created: 2023/11/9
```

**Review by josedahlquist — APPROVED** [review]

File: IPs/RSKIP412.md (hunk 1 above)
Comment (nagarev) [reply IPs/RSKIP412.md]: done

File: IPs/RSKIP412.md (hunk 2 above)
Comment (nagarev) [reply IPs/RSKIP412.md]: done

**Pull request merged**